### PR TITLE
Rest Countries (Independent Publisher)

### PR DESCRIPTION
**Pull request by siddharth-vaghasia**
---
This is a custom connector to get all the countries lists and its relevant information using an open-source API restcountries.com
This would be useful where we need to create a dropdown to select the country and get is currency, language, timezone, etc also. 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

Screenshot of actions in Flow

image

Screenshot of  Test operations section within the Custom Connector UI

image

Let me know if any changes are required.

### CODE REVIEW

**Comment by natalie-pienkowska** [comment]
Thank you @siddharth-vaghasia for submitting your first Independent Publisher connector! Given the holidays, we will start validating your connector in the next 2 weeks.

File: independent-publisher-connectors/RestCountries/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,795 @@
+{
+    "swagger": "2.0",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/RestCountries/apiDefinition.swagger.json:2]: @siddharth-vaghasia Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

File: independent-publisher-connectors/RestCountries/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,795 @@
...
+    "consumes": [],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/all": {
+            "get": {
+                "responses": {
+                    "200": {
+                        "description": "200 OK",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/RestCountries/apiDefinition.swagger.json:26]: @siddharth-vaghasia Can it be just "OK", instead of "200 OK"?

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/RestCountries/apiDefinition.swagger.json (hunk 1 above)
Comment (siddharth-vaghasia) [reply independent-publisher-connectors/RestCountries/apiDefinition.swagger.json]: @sriyen-msft  - thanks for the review, can you please check now..I had fixed and commited again..

File: independent-publisher-connectors/RestCountries/apiDefinition.swagger.json (hunk 2 above)
Comment (siddharth-vaghasia) [reply independent-publisher-connectors/RestCountries/apiDefinition.swagger.json]: yes...I have updated this..

**Comment by siddharth-vaghasia** [comment]
@sriyen-msft  - review comments have been implemented...please check

**Comment by sriyen-msft** [comment]
@siddharth-vaghasia We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps. Thanks.

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
@siddharth-vaghasia Apologies for the delay in response. 

Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Pull request merged**